presto-main/src/main/java/com/facebook/presto/server/ServerConfig.java

```diff
@@ -22,6 +22,8 @@
 
 public class ServerConfig
 {
+    private boolean resourceManager;
+    private boolean resourceManagerEnabled;
```

Comment:
can you help me understand the meaning of these vars : `isResourceManager` is for the node and `isResourceManagerEnabled` is more of a global var?

Reply:
`isResourceManager` is for the node to identify itself as a resource manager.  `isResourceManagerEnabled` is for the cluster, and is used to determine if certain interactions with a resource manager should be expected.  It should be an illegal state if `isResourceManager == true && isResourceManagerEnabled == false` for a node.